components/script/dom/cssstylerule.rs

```diff
@@ -71,4 +76,25 @@ impl CSSStyleRuleMethods for CSSStyleRule {
             )
         })
     }
+
+    // https://drafts.csswg.org/cssom/#dom-cssstylerule-selectortext
+    fn SelectorText(&self) -> DOMString {
+        let guard = self.cssrule.shared_lock().read();
+        let stylerule = self.stylerule.read_with(&guard);
+        return DOMString::from_string(stylerule.selectors.to_css_string());
+    }
+
+    // https://drafts.csswg.org/cssom/#dom-cssstylerule-selectortext
+    fn SetSelectorText(&self, value: DOMString) {
+        let namespaces = Namespaces::default();
```

Comment:
(for future readers, there is now an issue here: https://github.com/w3c/csswg-drafts/issues/1511)